.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         # We do this instead of setting --default-tf-version because setting
         # that flag starts the download asynchronously so we'd have a race
         # condition.
-        TERRAFORM_VERSION: 0.13.3
+        TERRAFORM_VERSION: 0.13.5
```

Comment:
might be worth updating this to 0.13.7 while here to address HCSEC-2021-12.
https://discuss.hashicorp.com/t/terraform-updates-for-hcsec-2021-12/23570